staff/mgarciarojas/Javascript exercises/Hexadecimal.js

```diff
@@ -0,0 +1,37 @@
+//Hexadecimal
+//Writes a function that convert a hexadecimal color, "blue" for example
+// "#0000FF" in its RGB representation rgb(0,0,255)". 
+//Give the function the name getRGB() and test it with this code
+
+function getRGB(hexaColor){
+   var hexadecimal=hexaColor;
+   var rgb="rgb (";
+   for (var i =1; i<7; i=i+2){
+    if (hexadecimal[i]=='F' && hexadecimal[i+1]=='F'){
+        rgb+="255,";
+       
+    }else{
+        rgb+="0,";
+    }
+   }
+    rgb=rgb.substr(0, rgb.length-1);
+   rgb+=")";
+   return rgb
+}
```

Comment:
Have you checked this function w/ some random Hexadecimal colors like → #68ccb3, #db958b #4259d8 ?
A clue 👇 
#68ccb3 → 68 (Hex Number) cc (Hex Number) b3 (Hex Number) 
We can use [`parseInt`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt) to convert numbers in different bases (radix)
68 (Hexadecimal Number) → parseInt(68, 16) → 104 (Decimal Number)